lib/optional_query_cache/version.rb

```diff
@@ -0,0 +1,11 @@
+require 'active_record'
```

Comment:
Yeah, @mloughran suggested that structure on Friday; I'll probably do exactly that this morning.